.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 yarn.lock
+dist/
```

Comment:
Add final newline. Just use `dist` without `/`.

I recommend installing an [editorconfig](http://editorconfig.org/) plugin for your text editor. Things like final newlines will then be added automatically.

Reply:
Yea, thanks for the hint, was just about time to give my dotfiles a refresh 👍